mlflow/server/js/src/experiment-tracking/components/MetricsPlotPanel.js

```diff
@@ -29,6 +30,8 @@ export const METRICS_PLOT_POLLING_INTERVAL_MS = 10 * 1000; // 10 seconds
 // A run is considered as 'hanging' if its status is 'RUNNING' but its latest metric was logged
 // prior to this threshold. The metrics plot doesn't automatically update hanging runs.
 export const METRICS_PLOT_HANGING_RUN_THRESHOLD_MS = 3600 * 24 * 7 * 1000; // 1 week
+const MAXIMUM_METRIC_DATA_POINTS = 100_000;
```

Comment:
Hi @dbczumar, could you please let me know why this setting is hard-coded? We want to visualise metrics with more than 100k saved points, but we hit this limitation.